.github/workflows/main.yml

```diff
@@ -235,10 +235,15 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu", "windows"] # We do not test conda for MacOS
+        include:
+          - os: ubuntu
+            shell: bash -el {0}
+          - os: windows
+            shell: powershell
```

Comment:
shouldn't we also include macos since you added it later on?

Reply:
We disable macos because conda support is corrupted in GA.